kotlinx-coroutines-core/jvm/src/DefaultExecutor.kt

```diff
@@ -43,6 +43,7 @@ internal actual object DefaultExecutor : EventLoopImplBase(), Runnable {
 
     @Suppress("ObjectPropertyName")
     @Volatile
+    @OptIn(ExperimentalStdlibApi::class)
```

Comment:
This is clearly a breaking change in the language, something suspicious is here and we cannot accept it as is.

Reply:
I reported it to `kotlin-design`. Fix of this issue is definitely not the part of migration master of kotlin to 1.9

Reply:
Well, it means that the master is broken right now, and that it is definitely the part of preparing master to 1.9